### PR TITLE
Increase timeout for OSD Integtests 

### PR DESCRIPTION
**Pull request by Divyaasm**
### Description
Increase timeout for OSD Integtests 
### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by prudhvigodithi — APPROVED** [review]

File: jenkins/opensearch-dashboards/integ-test.jenkinsfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ def agent_nodes = [
 
 pipeline {
     options {
-        timeout(time: 4, unit: 'HOURS')
+        timeout(time: 5, unit: 'HOURS')
```
Comment (gaiksaya) [inline jenkins/opensearch-dashboards/integ-test.jenkinsfile:38]: Strange! Why does OSD tests need 5 hours to run?

Comment (Divyaasm) [reply jenkins/opensearch-dashboards/integ-test.jenkinsfile:38]: Keeping it buffer for now Sayali, Wanted to check if the newly added tests are actually running. Currently the run is aborted due to timeout error.

Comment (gaiksaya) [reply jenkins/opensearch-dashboards/integ-test.jenkinsfile:38]: That timeout might be at the functional-test level

Comment (prudhvigodithi) [reply jenkins/opensearch-dashboards/integ-test.jenkinsfile:38]: There has been failing tests with pipeline timeout for OSD and components (Related to 2.11.1 release).
 https://build.ci.opensearch.org/blue/organizations/jenkins/integ-test-opensearch-dashboards/detail/integ-test-opensearch-dashboards/4572/pipeline, [https://build.ci.opensearch.org/blue/organizations/jenkins/integ-test-opensearch-dashboa[…]detail/integ-test-opensearch-dashboards/4548/pipeline/114](https://build.ci.opensearch.org/blue/organizations/jenkins/integ-test-opensearch-dashboards/detail/integ-test-opensearch-dashboards/4548/pipeline/114), want to allow increasing the time and run the tests.

Comment (prudhvigodithi) [reply jenkins/opensearch-dashboards/integ-test.jenkinsfile:38]: If this still timeouts, there needs a deep dive in https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/2.11/integtest.sh.

**Pull request merged**